### PR TITLE
add fatal method

### LINKED ISSUE #1: add Fatal log level

### PR DESCRIPTION
**Pull request by salman0ansari**
fix #1

### CODE REVIEW

**Comment by salman0ansari** [comment]
@milhamsuryapratama add `hacktoberfest-accepted` label to both PRs

**Comment by salman0ansari** [comment]
> @milhamsuryapratama add `hacktoberfest-accepted` label to both PRs

File: log.go [hunk 1]
```diff
@@ -95,3 +95,7 @@ func (l *Logger) Warn() *Event {
 func (l *Logger) Error() *Event {
 	return l.newEvent(ErrorLevel)
 }
+
+func (l *Logger) Fatal() *Event {
+	return l.newEvent(ErrorLevel)
```
Comment (milhamsuryapratama) [inline log.go]: I think this is a fatal level not an error level @salman0ansari

Comment (salman0ansari) [reply log.go]: oops sorry

Comment (milhamsuryapratama) [reply log.go]: fatal level must return `os.Exit(1)` so which terminates the program immediately

**Comment by milhamsuryapratama** [comment]
I think in line 75 the `newEvent` function, you need to add a new parameter in the form of a function that we will execute, the function parameter is not empty

```
func (l *Logger) newEvent(level Level, done func(string)) *Event {
        // if func is not nil
        // execute the function and return

	e := newEvent(l.w, level)
	if level != NoLevel {
		e.Str(LevelFieldName, level.String())
	}
	return e
}
```

cc @salman0ansari

**Comment by milhamsuryapratama** [comment]
thank you @salman0ansari

**Review by milhamsuryapratama — APPROVED** [review]

**Pull request merged**